src/environments/environment.ts

```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  baseURL: 'https://exercise-log-nest.herokuapp.com',
```

Comment:
remove this url from here 
as rule, env files contain secure data (like url, or toke, etc). it's bad practice to push this information to VCS. 
https://harrisonpim.com/blog/you-should-commit-your-env-files-to-version-control